### ISSUE TITLE
Ability to remotely drop client without shutting down or rebooting the PC.

### ISSUE THREAD

**Issue by bruzona** [Libki/libki-server#186]
**Is your feature request related to a problem? Please describe.**
It would be useful to be able to remotely drop a client for several scenarios:

1. Computers are being used for some sort of class or other purpose of indefinite duration.
2. A patron is having issues logging in, or in situations where for some reason SIP2 connectivity to the ILS were broken
3. Performing PC maintenance tasks, such as Windows updates etc.

**Describe the solution you'd like**
A "drop client" functionality from the WebGUI

**Describe alternatives you've considered**
Dropping each PC individually, one by one,  using the backdoor password.

**Additional context**

**Comment by kylemhall** [comment]
By "drop client", I think you mean the ability to remotely kill the Libki client in the same manner as the backdoor password. Is that correct?

**Comment by laurwls** [comment]
We use Ansible for remote updating of our Windows PCs however I have discovered that the playbook is unable to open the Windows update dialog box unless someone is logged into the LibkiClient.  I can kill the Libki Client process using Powershell however the desktop, and task manager are still locked.  (no desktop shown and error when opening task manager)
It would be great if we could remotely shutdown the Libki client and unlocking the task manager and desktop.  Will this ever be possible?

**Comment by bruzona** [comment]
Yes, by "drop client" we mean remotely centrally cause Libki client to exit, but not shut down or reboot the PC. This could be helpful for manual maintenance tasks on a group of PCs, or perhaps when a computer lab is needed for a class etc.
